cpimauth_settings.php

```diff
@@ -0,0 +1,276 @@
+<?php
+class cpimauth_settings_page
```

Comment:
Yes, this class (which creates the B2C settings page) is instantiated at the bottom of this file. Line 274.

---
In reply to: [71235409](https://github.com/AzureAD/active-directory-b2c-wordpress-plugin-openidconnect/pull/3#discussion_r71235409) [](ancestors = 71235409)